src/theme/components/footer.ts

```diff
@@ -0,0 +1,132 @@
+import { displayScreenreaderOnly, screenreaderOnly } from "./globalMixins";
+
+export const footerGray = "#54514A";
+
+const Footer = {
+  baseStyle: {
+    backgroundColor: footerGray,
+    minHeight: { base: "360px", lg: "420px" },
+    padding: { base: "15px 20px 100px", md: "15px 0", lg: "30px 0" },
+    position: "relative",
+    // The two main lists.
+    listsContainer: {
+      display: "flex",
+      flexDirection: { base: "column", xl: "row-reverse" },
+      alignItems: { base: "flex-end", xl: "flex-start" },
+      marginTop: "75px",
+      margin: { md: "0" },
+      right: { md: "6%" },
+      position: { md: "absolute" },
+    },
+    // The first list which contains the three lists of text links.
+    footerLinksList: {
+      fontSize: { base: "14px", md: "13px", lg: "14px" },
+      fontWeight: { lg: "400" },
+      lineHeight: { base: "18px", md: "23px", lg: "30px" },
+      marginRight: { xl: "70px" },
+      order: { base: "2", md: "1", xl: "2" },
+      textAlign: { base: "right", md: "left" },
+      width: { base: "100%", md: "auto" },
+      li: {
+        float: { md: "left" },
+        marginTop: "0 !important",
+        width: { md: "135px", lg: "155px" },
+        _first: {
+          width: { md: "110px" },
+          marginRight: { lg: "110px", xl: "50px" },
+        },
+        ":nth-child(2)": {
+          marginRight: { lg: "60px", xl: "0" },
+        },
+      },
+    },
+    // First link in every list displays in mobile only.
+    footerLinksInner: {
+      li: {
+        ...screenreaderOnly(true),
+        _first: {
+          ...displayScreenreaderOnly,
+        },
+      },
+    },
+    // List of social media icon links.
+    socialMediaList: {
+      order: "1",
+      margin: "20px 0 15px 0",
+      marginTop: { md: "30px", xl: "0" },
+      textAlign: { xl: "right" },
+      width: { xl: "280px" },
+      li: {
+        display: "inline-block",
+        width: "45px",
+        a: {
+          display: "block",
+          color: "#FFF",
+          fontSize: { base: "40px", md: "50px" },
+          lineHeight: "45px",
+          textDecoration: "none",
+          svg: {
+            width: "36px",
+            height: "36px",
+          },
+        },
+      },
+    },
+    // For all links in the footer.
+    a: {
+      color: "ui.white",
+      fontWeight: "400",
+      margin: "0 0.2rem",
+      textDecoration: "none",
+      _hover: {
+        color: "ui.white",
+        textDecoration: "underline",
+      },
+      _focus: {
+        backgroundColor: footerGray,
+        outline: "solid 2px var(--nypl-colors-ui-white)",
```

Comment:
For border styles, we've split this into two attributes so we could use the theme object: `border` and `borderColor`.  Maybe use the same approach here?